pyproject.toml

```diff
@@ -70,6 +70,10 @@ more-itertools = "^9.0.0"
 portalocker = "^2.6.0"
 seaborn = "^0.12.1"
 pyyaml = "^6.0"
+tensorboard = [
+    {version = "^2.11", python = "<3.8"},
```

Comment:
@eddiebergman do we still need to support 3.7?

Reply:
Probably not, we would likely want it to be `">=3.8`

Reply:
I've maintained Python 3.7 support due to library dependencies. Until the library drops 3.7 support, removing it here would lead to errors for NePS users on Python 3.7.

Reply:
Can we at least change it to `">=3.7"` then?

Reply:
Important to not that officially even Python have stopped developing with 3.7 in mind by it's [end-of-life date](https://devguide.python.org/versions/)